core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java

```diff
@@ -29,12 +30,23 @@
 	@Parameter(description="The corresponding resource identifier(s) to match")
 	private List<String> resource;
 	
+	@Parameter(description = "The types of resources to get the versions for", in = ParameterIn.QUERY)
```

Comment:
The `in` value is not required, please remove it. All parameters are being registered as query parameters by default.

Reply:
I'm following the open API spec, which states the `in` parameter is required. The default value for `in` is `ParameterIn.Default`
https://spec.openapis.org/oas/v3.1.0#parameter-object
![image](https://user-images.githubusercontent.com/61234783/126483079-190cbe0b-d469-4ddd-9e13-a887074b60a3.png)

Reply:
Yes, that is required in the generated OAS3 spec JSON output, but the value is being added automatically by the `springdoc` generator, so you don't have to add it explicitly. Please remove it to be consistent with the other annotations. 🙇

Reply:
Done